src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityDistillationTower.java

```diff
@@ -66,8 +71,8 @@ protected BlockPattern createStructurePattern() {
                 .aisle("XXX", "X#X", "XXX").setRepeatable(0, 11)
                 .aisle("XXX", "XXX", "XXX")
                 .where('S', selfPredicate())
-                .where('Y', statePredicate(getCasingState()).or(abilityPartPredicate(MultiblockAbility.EXPORT_ITEMS, MultiblockAbility.INPUT_ENERGY, MultiblockAbility.IMPORT_FLUIDS)))
-                .where('X', fluidExportPredicate.or(statePredicate(getCasingState())))
+                .where('Y', statePredicate(getCasingState()).or(abilityPartPredicate(ALLOWED_ABILITIES)))
+                .where('X', fluidExportPredicate.or(statePredicate(getCasingState()).or(abilityPartPredicate(MultiblockAbility.MAINTENANCE_HATCH))))
```

Comment:
We should do my idea for how to automatically apply maintenance predicate when possible. I can do this if you would like. This will be a lot of comments if I mark everything individually so I will just leave it to this one